### ISSUE TITLE
Fix a11y 

### ISSUE THREAD

**Issue by alexanderrudnik** [altence/lightence-admin#160]
**Describe the bug**
lighthouse shows 60% score for a11y

**To Reproduce**
1. Run chrome lighthouse

**Expected behavior**
a11y should be >90%

**Comment by alexanderrudnik** [comment]
Adding a11y for some of the pages.
List of pages with a11 < 90%:
Dashboard: 80%
Advanced forms: 73%
Personal info (profile): 88%
Autocompletes (ui components): 88%
Inputs (ui components): 80%
Selects (ui components): 80%
